gcc-arm-embedded-toolchain.cmake

```diff
@@ -84,14 +85,27 @@ set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 
 # Create mandatory flags for processor, toolchain etc.
 set(TOOLCHAIN_COMMON_FLAGS "-ffunction-sections -fdata-sections")
+
+set(CORTEX_M0_FLAGS -mcpu=cortex-m0 -mthumb -mabi=aapcs -mfloat-abi=soft)
+set(CORTEX_M3_FLAGS -mcpu=cortex-m3 -mthumb -mabi=aapcs -mfloat-abi=soft)
+set(CORTEX_M4_FLAGS -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfloat-abi=hard -mfpu=fpv4-sp-d16 -fno-strict-aliasing -fno-builtin --short-enums)
```

Comment:
Jeśli chodzi o flagi do M4 to te strict aliasing, builtin i short-enums tu nie pasują.